src/test_IndexGame.py

```diff
@@ -1,6 +1,21 @@
 import pytest
 import IndexGame
 
+#2.1
+def test_chooseBuildingPool(monkeypatch):
+    monkeypatch.setattr('builtins.input', lambda _: "BCH HSE SHP FAC MON")
+    expected = { 
```

Comment:
here to, pls change it to accept the different way of collecting user input: 1 building entry at a time

Reply:
Will change the test accordingly

Reply:
thanks